packages/InlineEditors/README.md

```diff
@@ -39,7 +39,6 @@ To learn more about how to use it read on our `.mdx` documentation file
 
 ## Links
 
-- [Storybook Showcase](https://paprika.highbond.com/?path=/story/forms-inlineediting--showcase)
-- [GitHub source code](https://github.com/acl-services/paprika/tree/master/packages/InlineEditing/src)
+- [Storybook Showcase](https://paprika.highbond.com/?path=/story/forms-inlineeditors--showcase)
```

Comment:
Actually I dont see any forms in storybook for `inline editors` or `inline editing`

Reply:
this is standard we should create one later, maybe just remove it for now...